apps/openmw/mwgui/dialogue.cpp

```diff
@@ -364,6 +364,7 @@ namespace MWGui
         bool sameActor = (mPtr == actor);
         mPtr = actor;
         mTopicsList->setEnabled(true);
+        mTopicsList->scrollToFirstItem();
```

Comment:
Best not scrollToFirstItem in the (!resetHistory && sameActor) case, since no new window was opened.